docs/backEnd/DBHandler.md

```diff
@@ -17,7 +17,9 @@
  along with Bolinho.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-# [DBHandler.py](https://github.com/HefestusTec/bolinho/blob/main/src/DBHandler.py)
+# DBHandler
```

Comment:
# Naming

The main topic (with one `#`) **NEEDS** to have the same name as the page. 

You can find / rename on the `mkdocs.yml` file, on the `nav` section.

``` yaml
nav:
    - Back End:
          - DBHandler: backEnd/DBHandler.md
```

# Hierarchy

* There can only be one `#`.
* Sub-topics **must** be `##` onwards